### PR TITLE
Simpler Stochastic Block Model 

### PR DESCRIPTION
**Pull request by CarloLucibello**
as discussed  in #288

### CODE REVIEW

File: src/randgraphs.jl [hunk 1]
```diff
@@ -28,7 +28,10 @@ function DiGraph(nv::Integer, ne::Integer; seed::Int = -1)
     return g
 end
 
-"""Creates an [Erdős–Rényi](http://en.wikipedia.org/wiki/Erdős–Rényi_model)
+doc"""
+erdos_renyi(n::Integer, p::Real; is_directed=false, seed::Integer=-1)
+
```
Comment (sbromberger) [inline src/randgraphs.jl]: We don't need `doc` preceding docstrings, and I think the function signatures are included in the autogenerated stuff. Can you double check this please?

Comment (CarloLucibello) [inline src/randgraphs.jl]:
```
help?> erdos_renyi
search: erdos_renyi

  Creates an Erdős–Rényi random graph with n vertices. Edges are added between pairs of vertices with probability p. Undirected graphs are created by default; use is_directed=true to override.

  Note also that Erdős–Rényi graphs may be generated quickly using erdos_renyi(n, ne) or the Graph(nv, ne) constructor, which randomly select ne edges among all the potential edges.
```

Comment (CarloLucibello) [inline src/randgraphs.jl]: Also, doc""" is needed if you want to use $ for latex inlining, otherwise julia will interpolate symbols into the string.

**Comment by CarloLucibello** [comment]
Is there something withstanding the merge of this?

**Comment by sbromberger** [comment]
Just one - can you build the docs (go into `doc/` and run the build.jl script) and then see what the resulting markdown looks like?

**Comment by CarloLucibello** [comment]
I rebuild the doc, and the function `stochastic_block_model` I added with this PR doesn't show up in any of the .md. The doc build system needs some small fix.

**Comment by sbromberger** [comment]
See https://github.com/JuliaGraphs/LightGraphs.jl/issues/289 :)

**Comment by CarloLucibello** [comment]
This is a mess of cimmits. I'll open a new PR once the doc build fix is merged

**Pull request closed without merging**